.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "lib/llvm/src"]
 	path = lib/llvm/src
 	url = https://github.com/llvm/llvm-project
+  ignore = dirty
```

Comment:
what's this do? should it be intended more?

Reply:
It makes `git status` ignore uncommitted changes in the submodule.  I find this useful since `make libs` applies some patches to the LLVM code.  I'll fix the indent.

Reply:
@kulibali ping (indent)